src/sentry/static/sentry/app/utils/discover/fieldRenderers.tsx

```diff
@@ -296,13 +295,21 @@ const SPECIAL_FUNCTIONS: SpecialFunctions = {
 
       const palette = new Array(10).fill(theme.purpleDarkest);
       const score = Math.floor((userMisery / Math.max(uniqueUsers, 1)) * palette.length);
-      const miseryLimit = parseInt(userMiseryField.split('_').pop(), 10);
-      const title = `${userMisery} out of ${uniqueUsers} unique users waited more than ${4 *
-        miseryLimit}ms`;
+      const miseryLimit = parseInt(userMiseryField.split('_').pop() || '', 10);
+      const title = tct(
+        '[effectedUsers] out of [totalUsers] unique users waited more than [duration]ms',
+        {
+          effectedUsers: userMisery,
```

Comment:
I *think* it's affected?

```suggestion
        '[affectedUsers] out of [totalUsers] unique users waited more than [duration]ms',
        {
          affectedUsers: userMisery,
```